test/extended/apiserver/resiliency.go

```diff
@@ -0,0 +1,89 @@
+package apiserver
+
+import (
+	"context"
+	"fmt"
+	"strings"
+	"time"
+
+	"github.com/onsi/ginkgo"
+	"github.com/onsi/gomega"
+	configv1 "github.com/openshift/api/config/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/uuid"
+	"k8s.io/apimachinery/pkg/util/wait"
+	"k8s.io/kubernetes/test/e2e/framework"
+	e2eskipper "k8s.io/kubernetes/test/e2e/framework/skipper"
+
+	"github.com/openshift/origin/pkg/monitor"
+	"github.com/openshift/origin/pkg/monitor/monitorapi"
+	"github.com/openshift/origin/test/extended/single_node"
+	exutil "github.com/openshift/origin/test/extended/util"
+	"github.com/openshift/origin/test/extended/util/disruption"
+	types "k8s.io/apimachinery/pkg/types"
+)
+
+func forceApiRollout(oc *exutil.CLI) {
+	redeploymentReason := fmt.Sprintf(`{"spec":{"forceRedeploymentReason":"resiliency-test-%v"}}`, uuid.NewUUID())
+
+	_, err := oc.AdminOperatorClient().OperatorV1().KubeAPIServers().Patch(context.Background(), "cluster", types.MergePatchType,
+		[]byte(redeploymentReason), metav1.PatchOptions{})
+	gomega.Expect(err).NotTo(gomega.HaveOccurred())
+}
+
+func eventsContain(events monitorapi.Intervals, message string) bool {
+	for _, event := range events {
+		if strings.Contains(event.Message, message) {
+			return true
+		}
+	}
+
+	return false
+}
+
+var _ = ginkgo.Describe("[Conformance][sig-sno][Serial] Cluster", func() {
```

Comment:
@sttts any idea how should I disable the cluster monitor synthetic events test?
it seems like the infra runs it for every test

Reply:
@mfojtik

Reply:
I understand it to not be critical, as it will just mark flakiness and not fail the build